documentation/services_master_puppetserver.markdown

```diff
@@ -45,10 +45,25 @@ Puppet Server uses a Jetty-based web server embedded in the service's JVM proces
 
 The web server's settings can be modified in [`webserver.conf`](./config_file_webserver.html). You might need to edit this file if you're [using an external CA][external_ca] or running Puppet on a non-standard port.
 
+### Puppet Master Service
+
+Puppet Server includes a "master" service that provides the basic functions as
+Rack and WEBrick masters, using the same input and output formats.  See
+[Puppet V3 HTTP API](/puppet/latest/reference/http_api/http_api_index.html#puppet-v3-http-api)
+for more information on the basic APIs.  Puppet Server's master service
+provides some additional APIs that the Rack and WEBrick Puppet masters do not.
+
+- For docs on the Puppet Server-specific APIs hosted by the master service, see:
+
+    - [The `environment_classes` endpoint](./puppet/v3/environment_classes.markdown)
```

Comment:
:+1: to this section. I'll add the `static_file_content` endpoint here as well, and a similar link from the `puppet` repo's HTTP API index.